app/src/androidTest/java/com/github/se/polyfit/ui/screen/PostInfoScreenContentTest.kt

```diff
@@ -53,11 +55,8 @@ class PostInfoScreenContentTest {
 
     every { mealViewModel.meal } returns MutableStateFlow(Meal.default())
     every { mealViewModel.isComplete } returns MutableStateFlow(true)
-    every { viewPostViewModel.posts } returns MutableStateFlow(posts)
```

Comment:
i think because you deleted all references to `posts`, you can delete the variable on line 54